config/locales/en.yml

```
@@ -235,6 +251,18 @@ en:
       cta: Pay hiring fee
       subject: I hired a developer via RailsDevs
       title: Have you hired %{developer}?
+    hiring_invoice_requests:
+      create:
+        success: Thank you! Joe will reach out to you soon.
+      new:
+        billing_address: Billing address
+        description: Congrats on your new hire!
+        developer_name: Developer's name
+        feedback: Feedback (optional)
+        feedback_help: Please let us know how we can improve RailsDevs or anything you'd like to share.
+        requesting: Requesting...
+        submit: Submit
```

```suggestion
        request: Request
```

To match the developer form. Make sure to change view, too.